common/src/main/java/io/netty5/util/concurrent/AsynchronousResult.java

```diff
@@ -84,18 +86,18 @@
      * @return the result of this operation, if completed successfully.
      * @throws IllegalStateException if this {@code Future} or {@link Promise} has not completed yet.
      */
-    V getNow();
+    @Nullable V getNow();
```

Comment:
Is that acceptable to place nullness annotations right before the return type?
IMHO, it is more clearer because after all it is the return type that is notNull or nullable.